eng/targets/Imports.targets

```diff
@@ -423,4 +423,51 @@
       <EnableNgenOptimization>true</EnableNgenOptimization>
     </PropertyGroup>
   </Target>
+
+  <!--
+    Override the Arcade PackageReleasePackages target so that we can avoid changing The
+    Microsoft.CodeAnalysis.Analyzers version in the Roslyn package nuspec files.
```

Comment:
How do we avoid having to copy/paste this entire targets here, since this is going to be a maintinence problem? It looks like _BuildPackages is an input to the target, can we do a BeforeTarget PackageReleasePackages and rip out just the element we want like you're doing?